.github/workflows/nightly-release-test.yml

```diff
@@ -22,9 +22,9 @@ jobs:
       - name: Install packages
         run: |
           pip install --upgrade pip
-          pip install -U --pre --extra-index-url https://pypi.org/simple -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple .[test,default]
+          pip install -U --pre --extra-index-url https://pypi.org/simple -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple .[default,test,test-extras]
```

Comment:
One other unintended consequence of using `test-extras` is that `pytest-randomly` will also be installed, which will result in randomizing the test order for all CI runs.

We currently run with `pytest-randomly` in a scheduled job to avoid noise as `pytest-randomly` is really mostly for helping detect problems with the test suite itself rather than the underlying tested code.

The "safe" thing to do is remove the `test-extras` and just explicitly add `pytest-mpl` to the install line.